test/Libraries/PackageManagerTests/PackageLoaderTests.cs

```diff
@@ -472,6 +473,35 @@ public void GetOwnerPackageReturnsNullForInvalidFunction()
             Assert.IsNull(foundPkg);
         }
 
+        /// This test is added for this task: https://jira.autodesk.com/browse/DYN-2101. 
+        /// A followup task is added https://jira.autodesk.com/browse/DYN-2120 to refactor the approach to this solution.
+        /// This test needs to be modified in that case. 
+        [Test]
+        public void PackageLoadExceptionTest()
```

Comment:
👍

Reply:
can you add category as well:
`TechDebt`
https://github.com/DynamoDS/Dynamo/blob/8d4757760aa108a2aa446d42e35afd3971678369/test/DynamoCoreWpfTests/PublishPackageViewModelTests.cs#L157